crates/teloxide/src/dispatching/dialogue/get_chat_id.rs

```diff
@@ -1,20 +1,23 @@
-use crate::types::{CallbackQuery, ChatId, Message, Update};
+use crate::types::{
+    BotCommandScope, CallbackQuery, Chat, ChatId, ChatJoinRequest, ChatMemberUpdated, Message,
+    MessageCommon, Recipient, ResponseParameters, TargetMessage, Update, UpdateKind,
+};
 
-/// Something that may has a chat ID.
+/// Something that may have a chat ID.
 pub trait GetChatId {
     #[must_use]
     fn chat_id(&self) -> Option<ChatId>;
 }
 
-impl GetChatId for Message {
+impl GetChatId for CallbackQuery {
     fn chat_id(&self) -> Option<ChatId> {
-        Some(self.chat.id)
+        self.message.as_ref().map(|mes| mes.chat.id)
     }
 }
 
-impl GetChatId for CallbackQuery {
+impl GetChatId for MessageCommon {
```

Comment:
`MessageCommon` isn't used directly too, so `GetChatId` will be used rarely (or won't at all)

Reply:
That is true. But is it a problem to have this trait implemented for that anyway? See comment below